Obleak.Fody/ModuleWeaver.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Mono.Cecil;
+
+namespace Obleak.Fody
+{
+    public class ModuleWeaver
```

Comment:
This is what Fody itself uses to do the weave - it's standard across all fody extensions